packages/ember-engines/node_modules/.bin/ember

```diff
@@ -0,0 +1 @@
+../../../../node_modules/ember-cli/bin/ember
```

Comment:
Same as above, seems like this file should not be committed (I would expect `yarn install` to regenerated it as needed).